hiss/application/models.py

```diff
@@ -286,3 +288,48 @@ def clean(self):
             raise exceptions.ValidationError("First name can't contain any numbers")
         if any(char.isdigit() for char in self.last_name):
             raise exceptions.ValidationError("Last name can't contain any numbers")
+
+
+class WaveManager(models.Manager):
+    def next_wave(self, dt: datetime.datetime = timezone.now()):
```

Comment:
See comment regarding using `django.utils.timezone` instead of `datetime`

```suggestion
    def next_wave(self, dt: timezone.datetime = timezone.now()):
```